rust/chains/hyperlane-fuel/src/validator_announce.rs

```diff
@@ -1,31 +0,0 @@
-use async_trait::async_trait;
-
-use hyperlane_core::{
-    ChainResult, HyperlaneChain, HyperlaneContract, HyperlaneDomain, ValidatorAnnounce, H256,
-};
-
-/// A reference to a ValidatorAnnounce contract on some Fuel chain
-#[derive(Debug)]
-pub struct FuelValidatorAnnounce {}
-
-impl HyperlaneContract for FuelValidatorAnnounce {
-    fn address(&self) -> H256 {
-        todo!()
-    }
-}
-
-impl HyperlaneChain for FuelValidatorAnnounce {
-    fn domain(&self) -> &HyperlaneDomain {
-        todo!()
-    }
-}
-
-#[async_trait]
-impl ValidatorAnnounce for FuelValidatorAnnounce {
```

Comment:
I think we'll need this file when implementing https://github.com/hyperlane-xyz/fuel-contracts/issues/74, we should probably keep it even if it's not used atm?